commands/bridge.js

```diff
@@ -75,6 +78,7 @@ module.exports = function () {
 		const channel = channels.find(channel => channel.id == message.channelId);
 		if (!channel) return;
 		if (channel.name.endsWith('-private')) return;
+		if (!this.mqttClient) return;
 
 		if (message.cleanContent) {
 		    let displayName = message.author.displayName.replace("/", "");
```

Comment:
please remove all changes to this file - the check seems unnecessary since `onMqttMessage` will never be called if `mqttClient` isn't setup properly